.github/workflows/build-and-deploy-docker.yml

```diff
@@ -68,97 +68,102 @@ jobs:
         id: docker_image
         shell: bash
         run: echo "::set-output name=tag::${{ env.DOCKER_SHA }}"
```

Comment:
This should be changed soon https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/